cmd/kk/pkg/core/connector/host.go

```diff
@@ -65,6 +66,14 @@ func (b *BaseHost) GetInternalAddress() string {
 	return b.InternalAddress
 }
 
+func (b *BaseHost) GetInternalIPv4Address() string {
+	return strings.Split(b.InternalAddress, ",")[0]
+}
+
+func (b *BaseHost) GetInternalIPv6Address() string {
+	return strings.Split(b.InternalAddress, ",")[1]
```

Comment:
Same as the above, if the ipv6 address is not configured for the hosts, I think it should be reported `panic`.